### ISSUE TITLE
Crelu should have axis

### ISSUE THREAD

**Issue by botev** [tensorflow/tensorflow#15619]
Currently, the `tf.nn.crelu` activation concatenates along the last axis. This would work fine for dense layers and conv layers where the `data_fromat=channels_last`, but this would be incorrect if invoked on the widely used `data_format=channels_first` for conv layers on the GPU.

**Comment by facaiy** [comment]
Yes, it seems that those activation functions of `tf.nn` are designed for `channel_last` data format only.

**Comment by bignamehyp** [comment]
@vincentvanhoucke

**Comment by bignamehyp** [comment]
We currently don't have anybody working on this. It would be great if you could help us by working on this and submitting a PR. Let us know if you need further clarification. Thanks!

**Comment by yongtang** [comment]
Added a PR #15836 for axis support.
